packages/playground/src/utils/get_nodes.ts

```diff
@@ -178,9 +178,9 @@ export const getQueries = (mixedFilters: MixedFilter): Partial<NodesQuery> => {
     retCount: true,
     nodeId: +mixedFilters.inputs.nodeId.value! || undefined,
     farmIds: mixedFilters.inputs.farmIds.value,
-    farmName: mixedFilters.inputs.farmName.value,
-    country: mixedFilters.inputs.country.value,
-    region: mixedFilters.inputs.region.value,
+    farmName: mixedFilters.options.farmName,
+    country: mixedFilters.options.country,
+    region: mixedFilters.options.region,
```

Comment:
Why was it removed from the inputs and added to the options?

The distinction between `inputs` and `options` is as follows:

- Inputs: Any input provided by the user, such as `input`, `select`, `autocomplete`, and so on.

- Options: Filters that can be applied to the inputs, such as the page number, table count, switches, and so on.